app/Http/Requests/People/ActivitiesRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Requests\People;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class ActivitiesRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
```

Comment:
Can you explain to me where we now check if we are authorized to perform a method?